example/test.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -euxo pipefail
+
+readonly dir="$(realpath "$(dirname $0)")"
+cd $dir
+
+git grep -h '^\$' ./README.md |
+  sed -e 's/^\$ //g' -e 's/"/\\"/g' |
```

Comment:
Can this be simplified with `sed -ne 's/^\$ //p' -e 's/"/\\"/g' README.md  | xargs -I {} bash -exc "{}"` ?

Reply:
Nice idea. However, using `p` and `g` sed commands in one place did not work for me. And I think that it may be more readable the way it is right now, even though there is a little bit of redundancy.